### PR TITLE
Table to markdown enhancement

### PR DESCRIPTION
**Pull request by Noy-Maimon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc#28729

## Description
Added to tableToMarkdown function an optional url_keys argument for keys to be as clickable urls 

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -1341,7 +1340,47 @@ def appendContext(key, data, dedup=False):
...
+    :type url: ``Union[List[str], str]``
+    :param url: the url of interest or a list of urls
+
+    :return: markdown format for clickable url
+    :rtype: ``str``
+
+    """
+    if url and isinstance(url, list):
+        return ['[{}]({})'.format(item, item) for item in url]
+    return '[{}]({})'.format(url, url)
```
[the remark this answers is not in the document]
Comment (Noy-Maimon) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1380]: Done

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -1341,7 +1340,47 @@ def appendContext(key, data, dedup=False):
...
+        data = [url_to_clickable_dict(item, url_keys) for item in data]
+
+    elif isinstance(data, dict):
+        data = {key: url_to_clickable(value) if key in url_keys else url_to_clickable_dict(data[key], url_keys)
+                for key, value in data.items()}
+
+    return data
+
+
+def url_to_clickable(url):
```
[the remark this answers is not in the document]
Comment (Noy-Maimon) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:1367]: Done

File: Packs/Base/ReleaseNotes/1_4_3.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CommonServerPython
+- added to **tableToMarkdown** an optional argument fo url fields to be clickable
```
[the remark this answers is not in the document]
Comment (Noy-Maimon) [reply Packs/Base/ReleaseNotes/1_4_3.md]: Done

**Comment by Noy-Maimon** [comment]
@roysagi Can you please go over the following release notes: https://github.com/demisto/content/blob/73cb9a0df4ac1277b72be81cc73fcb8658c47904/Packs/Base/ReleaseNotes/1_4_3.md
Thank you!

**Review by reutshal — APPROVED** [review]

**Pull request merged**